content/people/alumni/kristina-riemer.md

```diff
@@ -9,14 +9,14 @@ title = "Kristina Riemer"
 summary = "PhD Student"
 
 # Optional image to display on homepage (relative to `static/img/` folder).
-image_preview = "kristina-riemer.jpg"
+image_preview = "alumni/kristina-riemer.jpg"
 
 # Tags: can be used for filtering projects.
 # Example: `tags = ["ernest-lab", "white-lab"]`
-tags = ["white-lab"]
+tags = [alumni"]
```

Comment:
looks like we missed "

Reply:
Thanks for tracking that down @henrykironde!